### PR TITLE
Virtualization improvements

### PR DESCRIPTION
**Pull request by grisu48**
This commit merges serveral improvements done in the latest virtualization runs:

* Fix storage restart issues (old images persist in pool directory)
* Update spectre-meltdown checker
* Fix networking test
* Update record_info for ensure_online
* Whitelist some known dmesg messages

- Related tickets: https://progress.opensuse.org/issues/71473 https://progress.opensuse.org/issues/71797 https://progress.opensuse.org/issues/71890
- Needles: -
- Verification run: http://phoenix-openqa.qam.suse.de/t2938

### CODE REVIEW

**Review by pdostal — APPROVED** [review]
I agree with all of that 👍 Thank you @grisu48

File: tests/virtualization/universal/guest_management.pm [hunk 1]
```diff
@@ -19,6 +19,17 @@ use testapi;
 use utils;
 
 sub run {
+    ## Ensure every guest has <on_reboot>restart</on_reboot>
+    foreach my $guest (keys %virt_autotest::common::guests) {
+        my $filename = "$guest.xml";
+        assert_script_run("virsh dumpxml $guest > $filename");
+        if (script_run("! cat $filename | grep 'on_reboot' | grep -v 'restart'") != 0) {
+            record_info("$guest bsc#1153028", "Setting on_reboot=restart for $guest");
+            assert_script_run("sed -i '/\<on_reboot\>/c\  <on_reboot\>restart\</on_reboot\>/' $filename");
```
Comment (grisu48) [inline tests/virtualization/universal/guest_management.pm:28]: @pdostal Those two lines have been forgotten and have been added now.

**Comment by grisu48** [comment]
I would also invite @Julie-CAO to review the changes as at least the networking part is shared with you :-)

File: tests/virtualization/universal/guest_management.pm (hunk 1 above)
Comment (pdostal) [reply tests/virtualization/universal/guest_management.pm:28]: 👍

**Comment by Julie-CAO** [comment]
> I would also invite @Julie-CAO to review the changes as at least the networking part is shared with you :-)

About the network part, I took a general look, it should be no problem. When I come back from our long holiday(Oct 1 ~ Oct 9), I'll do some verification run. I'd not like to block your job. I'll let you know if I meet problem.

**Comment by grisu48** [comment]
> > I would also invite @Julie-CAO to review the changes as at least the networking part is shared with you :-)
> 
> About the network part, I took a general look, it should be no problem. When I come back from our long holiday(Oct 1 ~ Oct 9), I'll do some verification run. I'd not like to block your job. I'll let you know if I meet problem.

Thank you Julie. We wish you a nice holiday in the meantime :sun_with_face: Let us know if you encounter any problems when you come back :slightly_smiling_face:

**Pull request merged**